### PR TITLE
[5.2] Small fix for the dt- tag in content details list

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue # .

### Summary of Changes
The article info in articles is built as a definition list.
The definion term is the text for Details,
The definition description is a list of different items like author, publishing-date ad so on

If the user decides to hide the definition term then the dt-tag remains empty.
With this small PR the the details info is not empty but hides the text only for sighted users, screen readers still have the complete definition term. 

This PR is of type "better than nothing" 
because a more correct solution for the would be a definition list which writes for every element 
`<dt>Author</dt><dd>Mr. John Doe</dd>`

### Testing Instructions
Easiest test is any blog in joomla. 
Just set some detail info in the corresponding menu item to "show" 
Then show and hide the details 
No change in frontend for sighted users, but check the source code.

### Actual result BEFORE applying this Pull Request
If the Article Info Title is set on "hide". The dt is empty 
grafik

### Expected result AFTER applying this Pull Request
grafik

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: layouts/joomla/content/info_block.php [hunk 1]
```diff
@@ -23,8 +23,12 @@
...
     ) : ?>
         <dt class="article-info-term">
-            <?php if ($displayData['params']->get('info_block_show_title', 1)) : ?>
-                <?php echo Text::_('COM_CONTENT_ARTICLE_INFO'); ?>
+            <?php if (!$displayData['params']->get('info_block_show_title', 1)) : ?>
+                <?php echo '<span class="visually-hidden">' ?>
+            <?php endif; ?>
+            <?php echo Text::_('COM_CONTENT_ARTICLE_INFO'); ?>
+            <?php if (!$displayData['params']->get('info_block_show_title', 1)) : ?>
+                <?php echo '</span>' ?>
```
Comment (Quy) [inline layouts/joomla/content/info_block.php]:
```suggestion
                <?php echo '<span class="visually-hidden">'; ?>
            <?php endif; ?>
            <?php echo Text::_('COM_CONTENT_ARTICLE_INFO'); ?>
            <?php if (!$displayData['params']->get('info_block_show_title', 1)) : ?>
                <?php echo '</span>'; ?>
```

Comment (chmst) [reply layouts/joomla/content/info_block.php]: thx

**Comment by brianteeman** [comment]
> because a more correct solution for the would be a definition list which writes for every element

Why not fix it this way

**Comment by chmst** [comment]
Can do. Do you think it is worth?

**Comment by brianteeman** [comment]
> If a job is worth doing it is worth doing well. 

Assuming of course that it is the correct thing to do and wcag seems to suggest that it is the correct way and even without hiding the "Details" it is wrong. 

https://www.w3.org/WAI/tutorials/page-structure/content/#description-lists

**Comment by chmst** [comment]
I like this quote. So I make this a draft  for now.

**Comment by brianteeman** [comment]
> I like this quote. So I make this a draft for now.

its a very famous "english" quote. I am sure there will be a german equivalent

**Comment by chmst** [comment]
Reviewing the code for this details block I think it is better to leave as is and just add this PR. 
What we have is not wrong, it is one definiton term with multiple descriptions and in in this context it is ok. I will not change other layouts.

**Comment by dautrich** [comment]
I have tested this item :white_check_mark: successfully on e11afa7e196b11f3adc354dd56f6b98a8b89e76b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43729">issues.joomla.org/tracker/joomla-cms/43729</a>.</sub>

**Comment by fgsw** [comment]
I have tested this item :white_check_mark: successfully on e11afa7e196b11f3adc354dd56f6b98a8b89e76b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43729">issues.joomla.org/tracker/joomla-cms/43729</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43729">issues.joomla.org/tracker/joomla-cms/43729</a>.</sub>

**Comment by bembelimen** [comment]
I've moved this to 5.2, I agree it's a bug fix, but what could happen is, that suddenly now with templates not supporting the CSS class, the label could appear, although it's disabled.

**Comment by pe7er** [comment]
Thanks @chmst !

**Pull request merged**